app/src/main/res/layout/activity_set_price_summary.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Consider using just a `LinearLayout` here instead, or a `FrameLayout`, I think it's called, which contains just a single component. Simpler since you don't need `ConstraintLayout` yet, but I don't think it changes much.